proto/protovalidate/buf/validate/validate.proto

```diff
@@ -559,10 +628,30 @@ message FloatRules {
   //   ];
   // }
   // ```
-  repeated float example = 9 [(priv.field).cel = {
+  repeated float example = 9 [(predefined).cel = {
     id: "float.example"
     expression: "true"
   }];
+
+  // Reserved for predefined rules using extension numbers defined within the
+  // [Protobuf Global Extension Registry][1]. Extension fields in this range
+  // that have the (buf.validate.predefined) option set will be treated as
+  // predefined field constraints that can then be set on field options of other
+  // messages to apply reusable field constraints.
+  //
+  // [1]: https://github.com/protocolbuffers/protobuf/blob/main/docs/options.md
+  extensions 1000 to 99999;
+
+  // Reserved for predefined rules using extension numbers that are not
+  // explicitly assigned. Extensions in this range that have the
+  // (buf.validate.predefined) option set will be treated as predefined field
+  // constraints that can then/ be set on field options of other messages to
+  // apply reusable field constraints.
+  //
+  // When using randomly generated numbers, please use a high-quality source of
```

Comment:
I still don't understand why we are referencing random number generation in the docs - I've read the comments, but this doesn't seem like a good idea.

Reply:
OK, references to random number generation are just removed.